src/main/java/com/softwire/todos/SourceControlLinkFormatter.java

```diff
@@ -0,0 +1,47 @@
+package com.softwire.todos;
+
+import java.util.function.BiFunction;
+
+public class SourceControlLinkFormatter {
+    private final String baseUrl;
+    private final BiFunction<String, Integer, String> linkFormatter;
+
+    public SourceControlLinkFormatter(Config config) throws ConfigException {
+        if (config.getGitblitUrl() == null && config.getGithubUrl() == null) {
+            throw new ConfigException("One of --gitblit-url or --gitblit-url must be specified");
```

Comment:
You probably don't need both this AND the check below too?

Reply:
Yeah, oops.